test/config-luatest/instance_config_schema_test.lua

```diff
@@ -1037,7 +1037,7 @@ g.test_box_cfg_coverage = function()
         audit_format = true,
         audit_filter = true,
 
-        -- TODO: Will be added in the scope of gh-8861.
+        -- For now defaults are managed by options applier.
         metrics = true,
```

Comment:
The problem is not only in defaults (otherwise we would just add it into `ignore_default`). The option is expressed as a record in the declarative configuration, so it needs a bit special handling in the test. See how `wal.ext` is handled.

Reply:
Gleb points me that there are no `box_cfg` annotations in the `metrics` schema node, so my suggestion regarding handling it like `wal.ext` doesn't work. OK. Let's just move the `metrics = true` in `ignore` close to `log`, where the reason of ignoring is pretty same.

Reply:
Done.